README.md

```diff
@@ -246,6 +247,27 @@ module.exports = {
 ```
 <br />
 
+**<a name='conditions'>`conditions:`</a>**
```

Comment:
Remove the repeat documentation in the lines
https://github.com/testbetter/openapitest/blame/ead668a8c56bc7badc44f714172915b2a7da1c8c/README.md#L292 and https://github.com/testbetter/openapitest/blame/ead668a8c56bc7badc44f714172915b2a7da1c8c/README.md#L112

Reply:
As we discussed that we will keep `repeat` tag outside of `conditions` that will not break previous work but we will remove `repeat` from inside of the `conditions` tag. If I misunderstood please let me know. @mateusfreira @goeverest

Reply:
I believe misunderstood, but as this is already implemented lets sync today and decide what to do.